src/components/ReportContact.jsx

```diff
@@ -1,83 +1,84 @@
 import React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import styled from 'styled-components';
 import { ArrowBack } from '@material-ui/icons';
-import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
 import PropTypes from 'prop-types';
 
+import { 
+  Typography,
+  Button,
+} from '@material-ui/core';
 
-const styles = {
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+`;
 
-  container: {
-    display: 'flex',
-    alignItems: 'center',
-    marginBottom: 16,
-  },
+const ButtonStyled = styled(Button)`
+  && {
+  min-width: 46px;
+  width: 46px;
```

Comment:
@mishs Doesn't forcing to a `width` make `min-width` redundant?

Reply:
@schalkventer Removes redundant width- commit 6e51bf4